code/modules/mob/mob_grab.dm

```diff
@@ -382,7 +382,15 @@
 			user.visible_message("<span class='danger'>[user] devours \the [affecting]!</span>")
 			if(affecting.mind)
 				add_attack_logs(attacker, affecting, "Devoured")
-
+			if((istype(affecting, /mob/living/simple_animal/hostile/poison/bees))) //Eating a bee will end up damaging you
+				var/obj/item/organ/external/mouth = user.get_organ(BODY_ZONE_PRECISE_MOUTH)
+				mouth.receive_damage(1)
+				if(istype(affecting, /mob/living/simple_animal/hostile/poison/bees/syndi))
+					user.reagents.add_reagent("facid", rand(1, 5))
+				else
+					user.reagents.add_reagent("spidertoxin", 5)
```

Comment:
Prob best to use the `beegent` variable on `/poison/bees` here. This would mean that eating a bee which has mutagen in its sting would mutate you :D

Reply:
Like this?

Reply:
Looks fine. The way ingestion works is always a bit wonky looking